dracut/anaconda-lib.sh

```diff
@@ -214,7 +214,16 @@ set_neednet() {
     [ -z "$netroot" ] && >> /tmp/net.ifaces
 }
 
+wait_for_entropy() {
+    while [ "$(cat /proc/sys/kernel/random/entropy_avail)" -lt "10" ]; do
+        ENTROPY_AVAIL=$(cat /proc/sys/kernel/random/entropy_avail)
+        warn "Waiting for entropy ($ENTROPY_AVAIL) to be >= 10"
+        sleep 1
+    done
```

Comment:
Good point, and no it really didn't make things noticeably better. Found a better solution anyway.